### ISSUE TITLE
Troubles with docstrings and autocomplete of parameters in imported function 

### ISSUE THREAD

**Issue by Chreutz** [autocomplete-python/autocomplete-python#372]
Hello.

I have written a module with some functions, `audio.py`. It contains a function like

    measure_transfer_function(sr, resolution, dbFS, device, channel_in, channel_out,delaysamples=0)

If this module is imported with

    import audio

writing 

![image](https://i.imgur.com/CAmUjLS.png)

does not autocomplete any parameters of the function.

However, importing it as 

    import audio as a

then it autocompletes properly:

![image](https://i.imgur.com/ckkdEQO.png)

Watching the log, it could be a jedi issue. 

in the working example it writes:

    autocomplete-python: Deserealizing response from Jedi {"id": "24ded8c46fbe0ac2c948773afc5c6b60", "results": [], "arguments": "${1:sr}, ${2:resolution}, ${3:dbFS}, ${4:device}, ${5:channel_in}, ${6:channel_out}, delaysamples=${7:0}$0"}

and the non-working:

    autocomplete-python: Deserealizing response from Jedi {"id": "8482b8828ae7442fb9ae257b98fc524d", "results": [], "arguments": "$0"}

Any ideas as to go about this? Or am I doing something wrong in the module I'm writing? If I do 

    import numpy

it perfectly autocompletes for numpy functions with `numpy.<function>(<arguments)`.

**Comment by sadovnychyi** [comment]
Works for me when I have both files saved, so not sure about this. Is it possible that somehow the response was cached? If you reload your editor with audio module being written, is it still broken?

You could also try [jedi-11](https://github.com/autocomplete-python/autocomplete-python/tree/jedi-11) with updated jedi (git clone + npm install + apm link to set it up) – it contains lots of bug fixes from jedi part, but not all tests are passing yet so not everything might work.

**Comment by Chreutz** [comment]
I have tried restarted atom with both files saved and it's the same afterwards. But I don't know if jedi is also restarted in that case. 

It could seem like it's cached, yes, but the function has never been saved without any arguments, so if that were the case, wouldn't it show an older set of parameters, not none?

**Comment by sadovnychyi** [comment]
Yeah, seems like not a caching issue. Could you try it on a custom branch? Just use `npm install && apm link` from jedi-11 branch and it will link that version into Atom.

**Comment by Chreutz** [comment]
I can try that, but it will probably not be until tomorrow. Thank you for your quick responses :-)

**Comment by Chreutz** [comment]
Came back into the office today, and now it won't autocomplete even with `import audio as a`. Not sure what is going wrong.
